packages/expo-modules-core/android/src/main/java/expo/modules/core/logging/Logger.kt

```diff
@@ -29,7 +29,7 @@ class Logger(
       this.add(OSLogHandler(category))
     }
     if (options.contains(LoggerOptions.logToFile)) {
-      this.add(PersistentFileLogHandler(category, context))
+      this.add(PersistentFileLogHandler(category, context!!))
```

Comment:
could we add some error handling code here in case someone use logToFile without passing a context?